README.md

```diff
@@ -1,5 +1,49 @@
-# $${\color{red}UNDER---CONSTRUCTION}$$
-This repository is currently under construction, but will provide Jupyter Notebooks that demonstrate how to use the JWST Calibration Pipeline
+# JWST Pipeline Notebooks
```

Comment:
We should leave the under construction banner for now. We still can run execution

Reply:
Added it back in.